contracts/standard/arbitration/ArbitrableTransaction.sol

```diff
@@ -24,26 +24,44 @@ import "./TwoPartyArbitrable.sol";
      *  @param _hashContract Keccak hash of the plain English contract.
      *  @param _timeout Time after which a party automatically loose a dispute.
      *  @param _partyB The recipient of the transaction.
+     *  @param _amountOfChoices The number of ruling options available.
      *  @param _arbitratorExtraData Extra data for the arbitrator.
      */
-    function ArbitrableTransaction(Arbitrator _arbitrator, bytes32 _hashContract, uint _timeout, address _partyB, bytes _arbitratorExtraData) TwoPartyArbitrable(_arbitrator,_hashContract,_timeout,_partyB,_arbitratorExtraData) payable {
-        amount+=msg.value;
+    constructor(
```

Comment:
It it working now? Because the last time I checked, using constructor did not work with truffle tests?

Reply:
Yes, at least in my environment it is working fine. All tests pass. 
I'm using truffle v4.1.12.